extensions/ql-vscode/src/vscode-tests/factories/remote-queries/variant-analysis-history-item.ts

```diff
@@ -0,0 +1,40 @@
+import { faker } from '@faker-js/faker';
+import { VariantAnalysisHistoryItem } from '../../../remote-queries/variant-analysis-history-item';
+import { QueryStatus } from '../../../query-status';
+import { VariantAnalysisStatus } from '../../../remote-queries/shared/variant-analysis';
+
+export function createMockVariantAnalysisHistoryItem(
+  historyItemStatus: QueryStatus = QueryStatus.InProgress,
+  variantAnalysisStatus: VariantAnalysisStatus = VariantAnalysisStatus.Succeeded,
+  failureReason?: string,
+  resultCount?: number,
+  userSpecifiedLabel?: string
+): VariantAnalysisHistoryItem {
+  return ({
+    t: 'variant-analysis',
+    failureReason,
+    resultCount,
+    status: historyItemStatus,
+    completed: false,
+    variantAnalysis: {
```

Comment:
Can we re-use [`createMockVariantAnalysis`](https://github.com/github/vscode-codeql/blob/4b9db6a2981bf46577dc21cb3b68df78db41bf01/extensions/ql-vscode/src/vscode-tests/factories/remote-queries/shared/variant-analysis.ts#L12) here?

Reply:
YES! 🙌